src/components/Dropdown/index.jsx

```diff
@@ -128,6 +126,7 @@ const Dropdown = ({
       role="dropdown"
       theme="light"
       trigger={isControlled ? undefined : TRIGGERS[trigger]}
+      zIndex={99999}
```

Comment:
@goutham-subramanyam _a Let’s revert this change and proceed with the second approach of decreasing the table column z-index.